src/components/VideoList/VideoList.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styled from 'styled-components';
+import mockData from '../../youtube-videos-mock.json';
+
+const { items } = mockData;
+
+const VideoCardContainer = styled.div`
+  border: 1px solid #cccccc;
+  border-radius: 5px;
+  padding: 20px;
+  background-color: #e7e7fc;
+`;
+
+const VideoCard = ({ info }) => (
+  <VideoCardContainer>
+    <img alt="" src={info.thumbnails.medium.url} />
+    <div>
+      <h2>{info.title}</h2>
+    </div>
+    <div>
+      <h3>{info.description}</h3>
+    </div>
+  </VideoCardContainer>
+);
```

Comment:
Also, I recommend abstracting this component to its own file component. We could try to add more things to the VideoList or to the VideoCardContainer and they will be difficult to maintain in the same file. The only recommended time to have two components or two logics on the same file it's when you change one file and then the other one, but if that happens they are coupled and it also not very recommended.